examples/src/main/scala/org/apache/spark/examples/ml/ChiSquareTestExample.scala

```diff
@@ -52,9 +52,9 @@ object ChiSquareTestExample {
 
     val df = data.toDF("label", "features")
     val chi = ChiSquareTest.test(df, "features", "label").head
-    println("pValues = " + chi.getAs[Vector](0))
-    println("degreesOfFreedom = " + chi.getSeq[Int](1).mkString("[", ",", "]"))
-    println("statistics = " + chi.getAs[Vector](2))
+    println(s"pValues = ${chi.getAs[Vector](0)}")
```

Comment:
I think this is OK; anything more complex I might suggest breaking out the expression into a val.

Reply:
Ok